media-libs/alembic/alembic-1.7.7.recipe

```diff
@@ -79,6 +93,9 @@ BUILD()
 {
 	mkdir -p build
 	cd build
+
+	export ILMBASE_INC_DIR="`finddir B_SYSTEM_HEADERS_DIRECTORY`/OpenEXR"
```

Comment:
I did not check yet, but it looks like it should be:
```diff
-	export ILMBASE_INC_DIR="`finddir B_SYSTEM_HEADERS_DIRECTORY`/OpenEXR"
+	export ILMBASE_INC_DIR="`finddir B_SYSTEM_DIRECTORY`/$relativeIncludeDir/OpenEXR"
```

Reply:
I have built your recipe on x86_64 and x86 secondary arch. (Will try on x86 primary arch asap.)

* On x86_64 everything is OK: `100% tests passed, 0 tests failed out of 63`
* On x86 secondary arch `BUILD()` works fine but `INSTALL()` fails. The headers are installed in the correct directories (thanks to the above change), but we need to tell cmake where the libraries should be installed.

I think it won't be too hard to find the correct parameters to pass to cmake. I'll leave that to you, but I can try to help you if you wish.

Hmm, maybe we just need to add:
```
	export ILMBASE_LIB_DIR="`finddir B_SYSTEM_DIRECTORY`/$relativeLibDir/OpenEXR"
```

Reply:
The changes to make this recipe work for x86 secondary arch are not very complicated, so here they are, in case you change your mind.
If you don't wish to mark x86 secondary arch as tested, that's OK. I will approve your PR even if you keep the `?x86` :)

```diff
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
 
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
 
 PROVIDES="
```

```diff
 	cmd:abctree
-	lib:libalembic$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libalembic$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
```

```diff
 PROVIDES_devel="
 	alembic${secondaryArchSuffix}_devel = $portVersion
-	devel:libalembic$secondaryArchSuffix = $portVersion compat >= 1
+	devel:libalembic$secondaryArchSuffix = $libVersionCompat
 	"
```

```diff
 defineDebugInfoPackage alembic$secondaryArchSuffix \
 	$libDir/libAlembic.so.$portVersion \
-	$binDir/abcconvert \
-	$binDir/abcdiff \
-	$binDir/abcecho \
-	$binDir/abcechobounds \
-	$binDir/abcls \
-	$binDir/abcstitcher \
-	$binDir/abctree
+	$commandBinDir/abcconvert \
+	$commandBinDir/abcdiff \
+	$commandBinDir/abcecho \
+	$commandBinDir/abcechobounds \
+	$commandBinDir/abcls \
+	$commandBinDir/abcstitcher \
+	$commandBinDir/abctree
+
+PATCH()
+{
+	if [ -n "$secondaryArchSuffix" ]; then
+		sed -i \
+			-e "s|\(DESTINATION\) lib|\1 $relativeLibDir|;" \
+			-e "s|\(ConfigPackageLocation\) lib/cmake/|\1 $relativeLibDir/cmake/|;" \
+			lib/Alembic/CMakeLists.txt
+	fi
+}
 
 BUILD()
 {
 	mkdir -p build
 	cd build
 
-	export ILMBASE_INC_DIR="`finddir B_SYSTEM_HEADERS_DIRECTORY`/OpenEXR"
+	export ILMBASE_INC_DIR="`finddir B_SYSTEM_DIRECTORY`/$relativeIncludeDir/OpenEXR"
```